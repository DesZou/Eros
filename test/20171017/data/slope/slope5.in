1000
2019033 4330244
2225983 7103546
5002276 1889378
6941773 6191501
2264056 8940877
9038658 5618936
7126856 7456169
8376461 585608
3648284 2694603
8041582 46845
2802829 766718
580346 7408332
2377548 3423872
3437851 393500
1368092 3901315
659853 2174153
7354836 4253613
7102819 3587172
436830 7924505
1227008 6876465
8920507 7144659
7258654 2021800
6893948 3879519
561611 346357
5875530 4117683
5595879 6969464
1827935 6495672
7281959 6468549
9287179 2087629
4853781 7484452
7914544 7797444
2826765 4245267
3680070 1177956
4749121 3586367
6636632 5680869
1125942 3659663
5350404 6272119
6876358 9135257
1958783 1100603
2406300 837931
1204367 3971272
7091259 6076483
2711056 1941159
8989303 3781655
8461680 6334423
9054457 8508431
4775987 7622091
4293440 6337349
7039215 2102247
4098843 8578700
9927120 5321735
3468208 5154150
4451093 7494904
2354936 7576844
6626834 3586428
7396416 4846058
4286764 7534154
2454442 8383520
6446688 9517451
7233834 3920771
8660157 9652447
9012234 1301146
3574373 6285910
764006 3388833
9236879 7039864
1202608 1798025
9844744 1610685
4388781 7954678
3425326 8839190
175862 6175040
7189107 6531426
3454328 9461875
9047264 7136249
3565045 6858959
4756752 8657189
368769 4375112
4303536 4605860
2037509 552839
73222 7426983
1022527 934503
6415847 440662
8867264 4656376
8973119 1100234
3200877 8092416
5104909 3927441
3544063 8096501
4143716 660781
324099 4954757
6264940 605462
4189914 2715022
8635742 4290996
6085434 5844902
4183691 908261
7360812 4131988
9636310 3923197
805900 6494677
1573430 4867137
9205004 9194905
4332341 8390123
3588656 7260355
9307252 8302249
2682835 9522284
4974372 8587038
5895329 932192
3671694 3948000
8725020 2629151
2321932 5818744
2802583 6048734
6463374 7414757
9874494 470659
4383638 9379864
8963724 2989485
9561404 4670841
8862802 8182186
4764203 7549282
8541623 5198719
6249506 5618847
8313523 3651811
6535668 4809972
7395731 7696324
8304776 8733591
8355320 7715265
4391769 5654705
8211544 9363118
2418039 7515233
8710779 3629972
9580677 3064121
1167359 7972014
3409849 28991
3661908 9786477
9739616 1984749
4260289 4891508
9490903 7612466
6884830 473500
2471186 7040367
1522199 2305522
7896089 3517563
7509552 941748
5145094 9456927
665531 9699288
7694088 3904923
1245603 8365701
3638371 8417439
6233637 3267992
5486958 4034898
6507474 6776353
9424148 9507279
3799694 4755171
1830483 8003874
1667220 5670359
3442880 7228880
3051631 3836457
9515649 3644237
3630850 3455289
3289425 5930627
1630466 1466523
9857920 8715474
6742263 6189848
9305841 2190602
6157631 7459372
1339162 4283890
9577226 7418374
9514119 1040325
811230 1548298
536535 8528304
6494741 5456210
5610685 6795349
8549851 6831145
4538881 1410763
7438897 1933327
2866428 4442908
5783276 6026936
4555936 2125766
3671094 1825420
1252206 505274
7207051 2470184
785533 3967584
5612478 4382812
6367738 6812689
866290 11052
4699042 2840987
8510753 481359
7054666 3948608
5342873 4108082
101335 4579511
3827706 7588086
8776263 7440810
4557113 6597102
9321766 9943735
135456 3173666
2193818 3755760
7602379 9431502
1856088 6873946
4372909 5968996
3263275 4071537
7616265 545542
1520756 6418979
323833 5799693
4065829 3883088
7512939 8983813
3836613 2964049
7226014 3289646
5158896 516846
6272426 8280957
9834580 4962918
8413045 4826520
6898101 8030955
9967807 3878110
8177257 6161826
4890775 1105373
5333468 4271144
4148963 5441113
7608078 3015816
6429916 2061516
4593168 7258438
2386064 1793249
2880267 650455
1335134 250232
6816185 2474535
4977021 8069669
7402601 6634770
217974 4685232
4274047 6274283
9941925 846742
7811826 9393784
3085629 5607687
3241933 7142259
275649 132582
2680110 559371
9503115 1706061
1185866 3076449
302664 5820166
4865895 3710475
4325625 5114514
1662689 2582677
2765940 7292582
123046 7818363
4151650 7472809
2765369 5767344
6374335 2032056
4287922 3503258
1829853 4629794
9670588 4204692
1633249 651116
7117846 254058
6588484 7932750
8182164 5804129
2265043 1399476
670875 3615532
3673392 5320126
758440 9644792
7363279 6682689
6820553 8063457
8153593 926660
5088418 5777274
1194572 2470787
5668330 4860106
3235641 212847
315755 8865481
5614981 9816876
6865798 6318695
1552841 7053713
3000236 537606
4970098 4364076
7928806 9019259
32577 1319100
2017998 2962448
7122252 3761973
1883284 6257925
8341628 3577151
9719674 6102026
735885 214071
1837656 4074055
7593488 5027117
1568409 5226325
1501292 17508
6014012 9990466
7699417 8362710
8752852 1842192
3161956 2466989
3742472 8942262
8329966 483530
6566932 5015454
3508781 9328962
394054 5791654
8913449 3259996
1891842 7168671
9476928 7651221
8309566 4965841
7212195 7371932
2899463 7821429
9267287 396886
605306 9197241
5185972 5655515
9116363 4814016
9664617 9057527
5811010 8198047
2994294 9490858
5996089 7654497
7425201 4713000
4224671 5650820
2277826 890842
9983097 5033338
7492775 2866139
1161307 727080
992945 6175740
4441715 7076203
1836932 662465
889599 1467150
7315739 873314
8461778 7760213
5578690 3179594
1204435 5807221
7792410 2069659
7623677 3392153
4046488 6905256
8425184 5385183
50566 8047226
6775327 6479577
2938369 830028
2197488 8920333
8760954 7254286
1957023 106741
5541813 6314597
6651531 4695296
7922765 7779114
2949031 1969551
8728237 9473511
1330939 6550521
763800 6090077
3171617 7853703
1758627 7235073
1158344 1431800
898614 9484922
6937652 1239806
4696078 3303077
1302715 8929855
9515710 5761425
1705290 916438
7047721 3318915
8206559 6311503
343050 5666675
6949132 2072878
5181036 659469
3227770 9645161
5740850 1473329
9925347 7162474
4576657 78963
3666839 9696910
6470544 2974108
9444600 6032040
5920911 5289255
216062 8383742
2918589 7512901
9579444 461515
2555169 884826
1597497 8116540
4734354 946429
429376 6842611
6535743 86780
917777 9675271
4537639 8671446
3283077 3267458
755490 5671871
7953549 9133921
5466070 3270559
289068 3170583
4236995 2257811
8756601 375307
4325560 3953515
5581214 2810901
5151963 1236138
7074267 7540230
8076198 6653961
7695236 7135005
52836 8057588
8954622 2149010
953784 5122060
530246 8615695
5666710 8423075
8970219 6776241
3210056 8177219
7121807 631692
6475626 5470888
8794474 2565413
881646 4542529
9247047 8789749
4747673 1098675
8196100 8834157
6817284 9101398
4972182 1722476
8654621 4179382
1262851 2941514
9319715 4953491
5937895 8766929
48813 9694689
4945136 7900991
918564 6126153
5870177 6987846
424772 528806
115987 8047371
4168223 8273929
3160559 5016106
4331802 8921654
7794805 3071665
1678958 7648036
6565688 5235873
7052457 6205185
4912480 6269936
885930 4345132
1693223 8329
5466024 186937
895580 5766986
7756588 1211248
1865187 2666117
2817426 4788046
4558515 6932864
3879436 9705218
567254 6122485
137631 7721842
3007597 8377234
9142692 2347828
2475817 2705786
9459656 33429
1344467 1816771
6306572 1298197
4550853 448758
206022 3712745
4918767 3134994
8518175 1930104
7226836 1239049
1376019 1332575
8166538 9897153
6531188 5215725
374461 6145044
2671078 3399335
3088300 5856977
6155236 2560069
7008042 5021241
5587514 5867954
1370654 7475459
541194 5570064
3087236 18904
5800263 8554291
2130282 5878487
4982973 6052897
7220596 8166419
9222791 7617177
6519256 6953616
624919 3104776
9569506 2265417
583473 1454485
8850418 9795245
9230832 8305943
6387833 889117
9551040 7123609
5173760 1735792
502114 1518429
5118625 569456
3085548 8557270
4401932 401754
5752390 4560812
3933699 4471649
5207342 5434812
5726686 5068460
2709301 2366357
670472 8552531
4479039 1660285
9515007 7621862
9058055 5301226
4953432 8714855
6159897 4004233
3598393 3716683
1116118 6449850
1770590 871164
1044212 4389138
3041941 8836442
6249595 2773208
6471745 5680053
1291977 3531272
5011547 8029653
8638350 8717907
3279340 5864307
492753 9418781
2219284 1072061
5794665 7210622
6381190 9819989
8303621 6215618
8085514 1907923
3102885 7107305
957690 4032958
6771231 3702951
6213061 1328663
1334171 7929445
9438319 9105999
6293526 3633982
3000207 2803404
7575395 3604227
9946959 786023
1646885 4535735
9893360 7542897
9002071 9717140
7525700 8737065
6984517 7703428
5705138 1686600
3905552 797353
3739957 5018230
5539885 4682582
2707165 3751890
6462054 5316115
1906252 7594605
5916120 746909
2755514 9403087
9149043 6615652
3984370 1692017
7883773 8605835
5965395 5060546
2579856 2841963
8321488 562211
2575767 1816976
381809 7800682
1661909 4361274
8064719 6870370
5703391 7399134
6879644 8834121
6492828 5914403
5930008 7333073
2751180 2953957
2871069 8503968
2424318 2873269
7942420 8876624
8549400 5209431
783614 1054832
3540308 6009923
6914156 3602561
2337853 1374757
6669191 9336314
3815094 8814685
4926473 4618308
1178483 134783
7555138 6157886
3107838 1301844
6840596 8715706
8662707 2578903
6344183 6508670
7095365 8160057
5033846 9587013
7196855 9715805
325295 3519164
2073955 9176669
9463639 9212421
4160374 3500675
8557364 4816054
4826610 998865
6047138 3920091
4158507 2960634
2929167 8418416
8715851 2394086
6758015 1573225
6340430 4062130
1833992 3961386
2259974 3986349
7940566 7761038
3237657 8188892
4938664 9883364
7676670 5508855
2791762 5666158
7852931 8398570
3083577 9616993
2139594 4380246
2519514 4673639
6043847 4093325
4244420 1489184
4965443 7092377
980505 8849052
3939292 6051126
8595984 1006563
3772163 9844179
6133494 8096749
6548605 9805333
4375593 1849898
7989859 3736659
1898651 4350844
1239587 5708699
1688789 8454328
3859068 7058348
8385266 4987817
7736837 1087610
3438239 8065420
6716199 7505070
1693602 341141
7759774 4047907
8965584 8286697
4471548 887402
5970837 3404833
3861030 563526
5209712 6946894
2542164 866865
2315124 7249903
4129942 5667669
1475785 446972
3817418 5114260
8577320 1149791
4445223 4742016
5527906 5180534
9456821 4082532
4064474 2800275
9796946 5572793
2231047 5880700
310447 9189938
6923507 2092395
7699153 9607345
4676543 9294862
5298705 6789583
5299484 2008584
8155420 5464920
45395 7859310
6279539 6549196
977470 4911947
7071340 8869472
8643405 4369821
8014127 5191291
311426 5991638
8834874 9902393
4502389 6980188
4128478 3647467
5382481 8162825
4827072 3000725
1800533 3971116
9464408 7187823
1415380 6901254
3060085 7787877
4598709 4842957
6303287 4927894
387571 2626989
822932 1653849
7900318 2666690
7844052 2408883
9408985 6698783
643221 4524258
7584218 9082186
1779656 6366678
818670 1379645
1346993 6178482
7690098 1321575
9004395 8373712
6888282 7056809
2958946 9793210
2736558 1047642
768215 6993330
7238536 9789123
4020448 7006379
1395863 9122851
7548005 6749261
6059039 879533
3940516 8682228
1295607 9185669
6136365 6243250
5881226 6906182
7863143 8815305
1336918 99839
3381769 932018
6056432 7007793
8919842 7942937
5303550 8174163
924818 183134
74044 5201871
711694 9303574
4245623 9480706
8840096 5982600
9236117 5317440
4529189 3605680
7441163 3594957
3150926 9060159
41061 4473189
3059615 6444313
6267458 7073845
303474 9883861
2563413 1817567
7574133 9459265
9998822 4646778
4812584 8625628
6808654 4392208
2958944 2489434
4481279 3984670
6579911 4831908
1125754 3519512
4902362 4829397
2130902 6661603
8081001 6262205
3365440 1891416
3667739 8805730
9055349 7124295
5490791 5945082
3296586 9793266
6116764 6996851
3859620 2199054
6296665 2274795
4760577 3680070
4744470 973240
993005 8475448
9932442 4907489
702478 4752886
3313282 3582755
4620313 4788229
8104384 7374189
1648985 51553
3371674 1268836
3227055 4568312
6683431 1522144
1932301 2321962
2568405 1454189
4585083 6706128
3416550 5291723
5586194 4883378
1937902 2375279
8168963 1873678
80983 899584
2635409 6266392
2386141 1658714
3535762 4842741
4835219 3840910
1282675 9215167
7808348 3639392
9065265 4188293
860914 6072647
5357920 3936200
283874 5584241
3052584 6061438
3713678 1306685
378307 4158226
4779690 1170367
1026432 7602018
2383168 6869311
9673667 3585344
1435820 7704762
5616387 4420466
5385760 5154400
1704651 371251
530849 1352213
8805434 6657839
631816 3359600
991618 3441162
9767426 6576446
3153900 6303962
7776657 804480
7084243 9711784
5364873 7429608
1535525 8322932
5505940 5023329
293733 7700398
1893806 3612164
5779499 2938429
1402921 6599588
5531298 7490427
4659754 438467
344969 6792119
3139498 9033242
9429812 5402162
1988598 827285
5700872 9036195
663343 1222740
407554 1474740
5159371 3144235
4647984 6102530
1130643 5713699
9695542 7878688
3173283 544759
3280587 1701013
2515309 86552
9749945 7787097
6297270 9976490
5853793 5100892
2185239 3114757
6897972 7464447
3578749 1784372
4650811 8973337
2896809 9173369
877456 1297780
5065521 8385351
4817507 4927054
7858685 1081543
6257270 3918168
4091083 4037332
4196897 92474
5304364 4674814
3267919 5264652
8563973 6095755
9454719 304990
4846863 4930673
7636036 5386034
9347543 9684317
4557381 9008893
2116798 5868592
7004862 5369770
4032890 7627772
7094141 3186326
1146003 5286283
2793574 2438549
384103 5319257
4074863 1999084
8205990 2339989
5523365 846549
1336663 7907364
7092261 9139582
3765714 793118
5484195 488458
5860804 7909199
9410663 3858062
6264494 4211255
6704331 1897877
9156454 8491787
7124286 7251869
2143124 3018161
7715397 3401623
7094582 9925320
9057268 4288747
7801628 1977758
6649081 4859890
7637613 1442487
2922002 227195
9695904 3868749
7198738 7337031
444548 2921035
9956960 8963468
3012409 5380816
1003127 7038197
5709493 5484978
2408807 513568
9982272 5821945
9997267 9652511
7443064 6535776
4625009 814968
9945293 5727575
2406925 1904421
7441926 5910532
2888454 4110357
2827962 4178749
6521551 7351593
955608 8134671
3928176 2749367
2850848 9829307
4172291 125067
1979687 2321971
8659639 8183009
5781207 4678817
4275536 4796524
2729965 5718113
6920869 8511186
7509369 2419586
9155104 2307245
3706338 3535305
3565250 5735308
174175 7275054
4049052 2482800
7170815 3786717
8427095 4334672
903202 1186998
915373 6187413
5789437 9834062
3410414 3551775
8803226 4875371
9655804 1777086
2466774 3669004
7683919 7390777
2638817 4160478
106354 3217227
2696897 977096
8478852 5611809
4408656 3277153
3424824 4084957
3205177 7033844
7312726 4725526
7627007 6164099
9212471 8334165
1651830 3125185
8565708 9380634
3809608 9941910
1934360 4166741
5404785 8505119
6831538 4419950
1980612 9845932
4135236 2477885
7435876 3041600
7944718 8555026
6285515 5286164
1503460 6723375
93767 596245
5237035 6815433
694775 9616353
3898208 1495947
1131090 5313614
9142802 1510290
6321825 8906851
2733155 1199858
3539072 5066588
9811851 3386671
7646946 1549867
745429 7205317
1541898 130930
7940127 2689441
9737106 5806857
1994078 9301542
4467943 7855998
6091800 5319593
8080459 4577680
9047017 5039861
2547666 1601588
7079039 6377800
1166694 2718608
4073885 6350675
3170115 5244675
9025820 1224382
2444667 4426138
3811557 7726626
4393715 727068
8813304 1637518
484708 8005888
1816808 2511874
1137371 2791537
4430685 5615095
7423802 4612817
5817233 1478485
7400245 8914739
9875283 9823867
91093 2987487
1320327 6402747
8743800 3977810
3761700 7979226
6100687 7999548
5288116 8641557
2060767 4072192
316120 7101719
6235265 1904439
2066113 1548804
2860514 5075885
534712 9933554
9215108 7722979
5799703 6371562
234683 4807819
7018062 8318315
5372720 6920079
6592480 1154023
367969 3026187
6387683 584644
7647263 7489486
1566083 309980
5147279 6197469
2731263 2901118
259242 6829040
2112938 8362987
4545341 4945651
5084651 6201178
8512233 7538438
87230 8457076
8873380 3821181
2200881 3927256
5016630 9770209
5155816 3547928
6263101 1102829
2384665 2861371
3003946 4904338
5858192 3622585
2337346 9923059
2101549 4381698
6456222 5382762
3182817 6890615
2757675 5750145
5910709 4067695
2227498 769402
6627139 7102132
1636784 3509576
1168443 6895107
7624232 2584595
2937695 3382415
8922317 6258872
1637255 5484671
5010352 5463495
7071774 2582763
7878712 7636611
2484258 5264227
580253 4441317
2064808 2856937
8499819 4016921
3726422 7717490
5287686 1046144
6446696 4786536
3699872 8347820
5130668 3029803
8618692 649831
7687173 3666443
373455 3465517
5917292 1049808
9586034 8691757
2161386 4437683
488241 8416460
9922207 3574882
4047335 251298
8791742 3939967
2210496 3128058
