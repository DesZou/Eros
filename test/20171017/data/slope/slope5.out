3651888.000
